admin/app/components/solidus_admin/ui/tab/component.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::UI::Tab::Component < SolidusAdmin::BaseComponent
+  SIZES = {
+    s: %w[h-7 px-1.5 body-small-bold],
+    m: %w[h-9 px-3 body-small-bold],
+    l: %w[h-12 px-4 body-text-bold],
```

Comment:
Non-blocking nit: should we move `body-small-bold` to the list of common classes? Or do you foresee other sizes where we'll change the typo?

Reply:
Large (`l`) uses `body-text-small`, while the other two use `body-small-bold`, so I think it's easier to keep them here.

Reply:
Oh, I overlooked that :+1: